src/jmh/java/net/sf/jabref/benchmarks/Benchmarks.java

```diff
@@ -44,7 +44,7 @@ public void init() throws IOException, SaveException {
         Random randomizer = new Random();
         for (int i = 0; i < 1000; i++) {
             BibEntry entry = new BibEntry();
-            entry.setCiteKey("id" + i);
+            entry.setBibTeXKey("id" + i);
```

Comment:
According to [WikiPedia's BibTeX article](https://en.wikipedia.org/wiki/BibTeX), there is a bibtex field `key`, which is ...

> A hidden field used for specifying or overriding the alphabetical order of entries (when the "author" and "editor" fields are missing). Note that this is very different from the key (mentioned just after this list) that is used to cite or cross-reference the entry.

BibTeX key is defined as follows:

> In addition, each entry contains a key (Bibtexkey) that is used to cite or cross-reference the entry. This key is the first item in a BibTeX entry, and is not part of any field

 * Biber offers the `id` field and refers to the bibtex key as "citatoin key" ("Citation key aliases for the main citation key.")
 * [citation key is more common than cite key](http://www.googlefight.com/cite+key-vs-citation+key.php)

Since "citation key" is longer than the old Cite key", I will undo this renaming and keep `CiteKey`.